packages/reporter/src/commands/command.tsx

```diff
@@ -258,6 +259,80 @@ interface Props {
   groupId?: number
 }
 
+const CommandDetails = observer(({ model, groupId, aliasesWithDuplicates }) => (
+  <span className={cs('command-info')}>
+    <span className='command-method'>
+      <span>
+        {model.event && model.type !== 'system' ? `(${displayName(model)})` : displayName(model)}
+      </span>
+    </span>
+    {!!groupId && model.type === 'system' && model.state === 'failed' && <StateIcon aria-hidden className="failed-indicator" state={model.state}/>}
```

Comment:
Note for Monday -- this is where the fail-indicator class came from -- verify you don't need to re-add `vertical-align: middle`